### PR TITLE
Add guidance of using troubleshooting image

### PR DESCRIPTION
**Pull request by xiaomi7732**
Addressing #131

### CODE REVIEW

File: troubleshooting/AIK8sTroubleShooting/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM microsoft/dotnet:2.1-sdk AS build-env
+WORKDIR /app
+
+# Copy csproj and restore as distinct layers
+COPY *.csproj ./
+RUN dotnet restore
+
+# Copy everything else and build
+COPY . ./
+RUN dotnet publish -c Release -o out
```
Comment (karolz-ms) [inline troubleshooting/AIK8sTroubleShooting/Dockerfile]: Why not debug?

Just curious--it is for troubleshooting after all :)

File: troubleshooting/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Troubleshooting the Environment
+Kubernetes cluster configuration could be complex. To run the app with `Application Insights for Kubernetes` on does not always work for the first shoot. It is a challenge when it is not clear where the issue comes from - the configuration for Kubernetes, the build of the images or the library of `Application Insights for Kubernetes`.
```
Comment (karolz-ms) [inline troubleshooting/README.md]: .. does not always work on the first try

File: troubleshooting/AIK8sTroubleShooting/Dockerfile (hunk 1 above)
Comment (xiaomi7732) [reply troubleshooting/AIK8sTroubleShooting/Dockerfile]: I will keep it as is for now but I am totally open to change it to debug if necessary.
There is currently almost no user code for debugging per say. That might change later.

File: troubleshooting/README.md (hunk 2 above)
Comment (xiaomi7732) [reply troubleshooting/README.md]: Will fix this. Thanks!

**Pull request merged**